core/drivers/crypto/caam/include/caam_trace.h

```diff
@@ -279,6 +280,29 @@
 #define MP_DUMPBUF(...)
 #endif
 
+/* KEY */
+#if CAAM_DBG_TRACE(KEY)
+#define KEY_TRACE DRV_TRACE
+#if CAAM_DBG_DESC(KEY)
+#define KEY_DUMPDESC(desc)                                                     \
+	do {                                                                   \
+		KEY_TRACE("KEY Descriptor");                                   \
+		DRV_DUMPDESC(desc);                                            \
+	} while (0)
```

Comment:
Preferably move the line continuation symbols `\` to the left (align on the next tab stop after the "KEY Descriptor" line)

Reply:
I wanted to align with the style in the file but okay, I will fix